__tests__/index.test.js

```diff
@@ -10,33 +10,66 @@ beforeEach(() => seed(testData))
 afterAll(() => db.end())
 
 describe('GET /api/topics', () => {
-    test('responds with an array of all topic objects with SLUG and DESCRIPTION as properties', () => {
+    test('GET 200: responds with an array of all topic objects with SLUG and DESCRIPTION as properties', () => {
         return request(app)
-        .get('/api/topics')
-        .expect(200)
-        .then((response) => {
-            const body = response.body
-            expect(body.topics.length).toBe(3);
-            body.topics.forEach((topic) => {
-                expect(typeof topic.description).toBe('string')
-                expect(typeof topic.slug).toBe('string')
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                const body = response.body
+                expect(body.topics.length).toBe(3);
+                body.topics.forEach((topic) => {
+                    expect(typeof topic.description).toBe('string')
+                    expect(typeof topic.slug).toBe('string')
+                })
             })
-        })
     })
 })
 
-// create an app that in the model adds to the endpoints.json in a dynamic way
 describe('GET /api', () => {
-    test('responds with an object describing all available endpoints on API', () => {
+    test('GET 200: responds with an object describing all available endpoints on API', () => {
+        return request(app)
+            .get('/api')
+            .expect(200)
+            .then((response) => {
+                const body = response.body
+                console.log(body)
+                expect(body.endpointsData).toEqual(endpointsData)
+            })
+    })
+
+})
+
+describe('/api/articles/:article_id', () => {
+    test('GET 200: responds with the object of the queried article_id', () => {
         return request(app)
-        .get('/api')
-        .expect(200)
+            .get('/api/articles/1')
+            .expect(200)
+            .then((response) => {
+                const body = response.body.article
+                expect(body.article_id).toBe(1)
+                expect(typeof body.author).toBe('string')
```

Comment:
I would prefer to see this a little more certain of the data, yes you're asserting you have the right article id. What about the title or author?